src/server.py

```diff
@@ -372,8 +373,8 @@ def run_jobtypes_query():
 @app.route("/data/create_jobtypes/")
 @json_response
 def run_create_jobtypes_query():
-    # skipping b2g*, android*, mulet*
-    platforms = ['android-2-3-armv7-api9', 'android-4-2-x86', 'android-4-3-armv7-api11', 'osx-10-6', 'osx-10-10', 'windowsxp', 'windows7-32', 'linux32', 'linux64', 'windows8-64']
+    alljobs = jobtype_query()
+    platforms = list(set(job[0] for job in alljobs))
 
```

Comment:
hmm, it appears I messed up the landing on the master branch and never
actually merged it.  I am not sure how the previous pull request was merged
in:
https://github.com/dminor/ouija/commit/9efc6e7eb80b04a8a56ce8398e9ac1cae4422738

but not showing up.

On Wed, Sep 16, 2015 at 6:30 PM, Vaibhav Agrawal <notifications@github.com>
wrote:

> In src/server.py
> <https://github.com/dminor/ouija/pull/78#discussion_r39693706>:
>
> > @@ -372,8 +373,8 @@ def run_jobtypes_query():
> >  @app.route("/data/create_jobtypes/")
> >  @json_response
> >  def run_create_jobtypes_query():
> > -    # skipping b2g*, android*, mulet*
> > -    platforms = ['android-2-3-armv7-api9', 'android-4-2-x86', 'android-4-3-armv7-api11', 'osx-10-6', 'osx-10-10', 'windowsxp', 'windows7-32', 'linux32', 'linux64', 'windows8-64']
> > +    alljobs = jobtype_query()
> > +    platforms = list(set(job[0] for job in alljobs))
> >
>
> This was already changed and merged in #77
> <https://github.com/dminor/ouija/pull/77>
> @jmaher <https://github.com/jmaher> How come it is not present in the
> repository??
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dminor/ouija/pull/78/files#r39693706>.
>